src/scripts/blocks/router/router-helper.provider.js

```diff
@@ -0,0 +1,36 @@
+function routerHelperProvider($locationProvider, $stateProvider, $urlRouterProvider) {
+  'ngInject';
```

Comment:
If we do, we're registering the functions with angular in the same class that we're creating the function.  The pattern I've used for the dayjob is to bundle the classes/functions into a bucket and perform all the declarations for the angular module in that file. 

I have seen examples where developers perform the declaration for their module within the declaration file but it doesn't work with the go-foward with ng2.  You still have to export your components/injectables in a module file.  The practice I'm using here is closest to how you'll see ng code moving forward.